src/main/java/com/github/joelittlejohn/embedmongo/AbstractEmbeddedMongoMojo.java

```diff
@@ -102,19 +111,26 @@ protected IFeatureAwareVersion getVersion() {
             versionEnumName = "V" + versionEnumName;
         }
 
+
+        Feature[] features = new Feature[0];
+        try {
+            features = Stream.of(this.features.split(",")).map(String::trim).map(String::toUpperCase).map(Feature::valueOf).toArray(Feature[]::new);
```

Comment:
I'm not sure, but if the field `features` is not set and stays `null` won't this `this.features.split(",")` throw a NullPointer? Maybe `features` needs a default?

Reply:
I added as suggested a null check, unit test and documentation.